csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -170,12 +170,33 @@ public void AppendExecutionProvider_Nuphar(string settings = "")
         #endregion //ExecutionProviderAppends
 
         #region Public Methods
+
+        /// <summary>
+        /// (Deprecated) Loads a DLL named 'libraryPath' and looks for this entry point:
+        /// OrtStatus* RegisterCustomOps(OrtSessionOptions* options, const OrtApiBase* api);
+        /// It then passes in the provided session options to this function along with the api base.
+        /// Deprecated in favor of RegisterCustomOpLibraryV2() because it provides users with the library handle 
+        /// to release when all sessions relying on it are destroyed
+        /// </summary>
         public void RegisterCustomOpLibrary(string libraryPath)
```

Comment:
Can we mark it with [Obsolete] attribute?

Reply:
Done